src/js-transform.ts

```diff
@@ -151,6 +158,21 @@ export function jsTransform(js: string, options: JsTransformOptions): string {
         options.componentDir,
         options.rootDir));
   }
+  if (options.transformImportMeta) {
+    if (!options.filePath) {
+      throw new Error('Cannot perform importMeta transform without filePath.');
+    }
+    if (!options.rootDir) {
+      throw new Error('Cannot perform importMeta transform without rootDir.');
+    }
+    doBabel = true;
+    let relativeURL = relative(options.rootDir, options.filePath);
```

Comment:
Can you update the description for filepath and rootdir to say that they must both be absolute system paths?

Reply:
done